gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector-error/policy-studio-debug-inspector-error.component.html

```diff
@@ -17,9 +17,21 @@
 -->
 <table class="gio-table-light" attr.aria-label="Debug errors">
   <tbody>
-    <tr *ngFor="let item of errors">
-      <td class="key">{{ item.key }}</td>
-      <td class="value">{{ item.value }}</td>
+    <tr *ngIf="error.status">
+      <td class="key">Status</td>
+      <td class="value">{{ error.status }}</td>
+    </tr>
+    <tr *ngIf="error.message">
+      <td class="key">Message</td>
+      <td class="value">{{ error.message }}</td>
+    </tr>
+    <tr *ngIf="error.contentType">
+      <td class="key">Content Type</td>
+      <td class="value">{{ error.contentType }}</td>
+    </tr>
+    <tr *ngIf="error.key">
+      <td class="key">Key</td>
+      <td class="value">{{ error.key }}</td>
```

Comment:
why not iterating on the key of object `error` to display them in a `ngFor` way ?

Reply:
Because I can't order easily. And the default order will be alphabetical. 
That why I've change to have a clear order